examples/format_detection_openai_gpt.py

```diff
@@ -101,26 +116,24 @@ def predict_value_udf(s):
     openai.api_key = openai_key_broadcast.value
     openai.api_type = "azure"
     openai.api_version = "2023-05-15"
-    
+
     def predict_value(s):
         content = f"Please categorize the following value {s} based on its format into one of the following categories: IPv4, IPv6, MAC, NOT MATCHED. Please reply with just category name"
         response = openai.ChatCompletion.create(
```

Comment:
You are using the chat API, but you don't need the chat history. Use the Completion API instead, and provide max_tokens = 1